utils/build-script

```diff
@@ -839,8 +891,35 @@ the number of parallel build jobs to use""",
     for v in ['MAKEFLAGS', 'SDKROOT', 'MACOSX_DEPLOYMENT_TARGET', 'IPHONEOS_DEPLOYMENT_TARGET']:
         os.environ.pop(v, None)
 
+    if args.show_sdks:
+        swift_build_support.debug.print_xcodebuild_versions([
+            'iphonesimulator',
+            'appletvsimulator',
+            'watchsimulator',
+        ])
+
     check_call(build_script_impl_args)
 
+    if args.symbols_package:
+        symbols_package_path = os.path.abspath(args.symbols_package)
+        print('--- Creating symbols package ---')
+        print('-- Package file: {} --'.format(symbols_package_path))
+
+        if platform.system() == 'Darwin':
+            prefix = swift_build_support.targets.darwin_toolchain_prefix(
+                args.install_prefix)
+        else:
+            prefix = args.install_prefix
+
+        # As a security measure, `tar` normally strips leading '/' from paths
+        # it is archiving. To stay safe, we change working directories, then
+        # run `tar` without the leading '/' (we remove it ourselves to keep
+        # `tar` from emitting a warning).
+        if args.install_symroot:
+            os.chdir(args.install_symroot)
+        swift_build_support.tar.tar(source=prefix.lstrip('/'),
```

Comment:
I feel like the tar() function itself should be doing chdir() -- and back.  We have a `WorkingDirectory` class in utils/SwiftBuildSupport.py, by the way.